content/agents/overview.md

```diff
@@ -54,6 +54,15 @@ Following are all the supported installation methods:
 * `init_script` - An agent will be installed via a script that will run on the host when it gets created. This method is only supported for specific IaaS plugins.
 * `provided` - An agent is assumed to already be installed on the host image. That agent will be configured and started via a script that will run on the host when it gets created. This method is only supported for specific IaaS plugins.
 
+{{% gsNote title="Implications of Not Installing an Agent" %}}
+In some cases, you cannot or prefer not to install an agent on Cloudify-managed VMs. This could be due to a security restriction or the fact that a VM is a pre-baked closed appliance that you can't access or modify. 
+In such cases, you would use `install_method: none` to instruct Cloudify not to install an agent on the created VM. 
+However, you should be aware of certain implications: 
+
+* You will not be able to use certain plugins, specifically ones that assume execution on the agent's VM, i.e. ones that are configured with `executor=central_deployment_agent`. This includes Docker, Chef and Puppet plugins among others. To work around this you will need to run bash or Python scripts using the [Fabric plugin](../../plugins/fabric/) (e.g. invoke the Puppet client from a script instead of using the Puppet plugin). 
```

Comment:
Please use the directives for links, available here: https://github.com/cloudify-cosmo/docs.getcloudify.org/blob/3.3.1-build/content/cheatsheet.md